README.md

```diff
@@ -37,6 +37,236 @@ flutterfire --help
 dart pub global activate flutterfire_cli
 ```
 
+- FlutterFire CLI requires the Firebase CLI (`firebase-tools`) to be installed on your local machine, [follow these instructions](https://firebase.google.com/docs/cli) for installation.
+- If you're running on a windows machine, we highly recommend you install via npm (i.e. `npm install -g firebase-tools`). The standalone
+`firebase-tools` version can cause problems which you can read about [here](https://github.com/invertase/flutterfire_cli/issues/55#issuecomment-1316201478).
+
+## Documentation for `v0.3.0-dev.19`
+
+### Default setup
+
+You may run the following command to configure your Flutter application with Firebase. 
+
+```bash
+flutterfire configure
+```
+This method, at a minimum, will invoke two prompts of the user. A request to specify your Firebase project and
+the platforms you wish to configure (i.e. android, iOS, macOS, web & windows). 
+
+If you don't want any prompts, you can use the following command:
+
+```bash
+flutterfire configure --yes --project=<FIREBASE_PROJECT_ID>
+```
+
+FlutterFire CLI will auto detect the bundle ID for apple platforms via the `project.pbxproj` file, if there is no existing app associated with the bundle ID in your Firebase console, it will create one, and subsequently write the relevant `GoogleService-Info.plist` file to your project in the default location `ios/Runner/GoogleService-Info.plist` or `macos/Runner/GoogleService-Info.plist`. A similar process occurs on android, FlutterFire CLI uses the application ID in your `app/build.gradle` file and will generate a service file at the default location `android/app/google-services.json`.
```

Comment:
```suggestion
FlutterFire CLI will auto detect the bundle ID for apple platforms via the `project.pbxproj` file, if there is no existing app associated with the bundle ID in your Firebase console, it will create one, and subsequently write the relevant `GoogleService-Info.plist` file to your project in the default location `ios/Runner/GoogleService-Info.plist` or `macos/Runner/GoogleService-Info.plist`. A similar process occurs on Android, FlutterFire CLI uses the application ID in your `app/build.gradle` file and will generate a service file at the default location `android/app/google-services.json`.
```